src/components/DivElement.js

```diff
@@ -1,6 +1,13 @@
 const HTMLElement = require('./HTMLElement');
 
 // Define class here
+class DivElement extends HTMLElement{
+    constructor(content){
+        super(content);
```

Comment:
this constructor expects two params.   It just happens to work because HTMLElement constructor doesn't do much in this case